### PR TITLE
Add new csv type of report - per-task

### PR DESCRIPTION
**Pull request by fighterpoul**
This PR adds new CSV type of report - `per-task` - which displays a CSV document where each task has its details displayed in a row.

Details:
- [x] new csv-section: `per-task` or `per_task`
- [x] move timedelta_to_billable to report.common, as it is no longer used only by `per_day` report
- [x] integration test

### CODE REVIEW

**Comment by larose** [comment]
Thanks for this PR! The CI jobs failed because it was trying to use ubuntu-18.04 which is no longer supported by GitHub. I've updated it to ubuntu-22.04 in #102, but I also had to update all python dependencies because some of them were old and weren't compatible with python 3.8+.

Could you rebase your PR on master?

I'll aim to review your PR within the upcoming week.

**Comment by fighterpoul** [comment]
@larose done. Indeed I have started implementing this by using Python 3.10 but it was not working well as far as development is concerned. Great to get it updated :)

File: utt/report/details/view.py [hunk 1]
```diff
@@ -50,6 +53,26 @@ def render(self, output: Output) -> None:
 
         print(file=output)
 
+    def csv(self, output: Output) -> None:
+        if not self._model.activities:
+            print(" -- No activities for this time range --", file=output)
+            return
+
+        fieldnames = ["task", "project", "date", "duration", "type", "comment"]
```
Comment (larose) [inline utt/report/details/view.py:61]: In https://github.com/larose/utt/blob/e4143d6d120085e3e8b653053dced3837e655d48/utt/report/per_day/csv_view.py#L17 the usage of the plural form is applied to hours, projects and tasks. We should be consistent. Either we change to plural here or change to singular in `per_day/csv_view.py`. 

Also, let's arrange the columns in a more logical order, moving from general to specific. For instance, let's reorder the first three columns to be "date" "project" and "task" in that sequence.

**Review by larose — COMMENTED** [review]
Thanks! I've left one comment but otherwise it looks good to me!

**Comment by larose** [comment]
Thanks! Feel free to open a PR to put your real name [here](https://github.com/larose/utt/blob/e31c7340cd4c151b7a5bfa004b676ca72149978f/README.md?plain=1#L466) if you want.

**Pull request merged**